llvm/tools/llvm-jitlink/llvm-jitlink-executor/llvm-jitlink-executor.cpp

```diff
@@ -121,10 +159,11 @@ int main(int argc, char *argv[]) {
   int InFD = 0;
   int OutFD = 0;
 
+  std::vector<StringRef> TestOutputFlags;
+
   if (argc < 2)
     printErrorAndExit("insufficient arguments");
   else {
```

Comment:
`printErrorAndExit()` is noreturn. Is there a good reason to keep this `else` branch? Otherwise I'd drop it and  move the `TestOutputFlags` closer to the loop below. Also, I still need to update the help output.

Reply:
PR for it is here: https://github.com/llvm/llvm-project/pull/76689